### PR TITLE
#68 PillSelect Component complete and ready for review PR

### PR DESCRIPTION
**Pull request by clarknoah**
Component read for review

### CODE REVIEW

**Comment by ZakkMan** [comment]
@clarknoah Make sure you assign yourself the PR

File: client/src/components/PillSelect/PillSelect.stories.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  console.log(value);
+    }
+let pills = ['Parenting', 'Teens', 'Arts', 'Traveling', 'Primary School', 'Sports']
+
+ pills = pills.map(val=>{
+  let pill = <PillSelect
+    selected={true}
+    innerText={val}
+    selectedColor={'#FB836F'}
+    unselectedColor={'#FAFAFA'}
```
[the remark this answers is not in the document]
Comment (clarknoah) [reply client/src/components/PillSelect/PillSelect.stories.js]: This accepts all common CSS color formats, such as hex, string colors, and RGB/RGBA

File: client/src/components/PillSelect/PillSelect.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  justify-content: space-evenly;
+  align-items: center;
+  font-family: "Reem Kufi";
+  flex-wrap: wrap;
+}
+
+
+.pill-padding {
+  margin: 25px;
+}
```
[the remark this answers is not in the document]
Comment (clarknoah) [reply client/src/components/PillSelect/PillSelect.css]: I've removed this CSS class

File: client/src/components/Pill/Pill.stories.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  .add("Community People Pill (Toggle)", () => (
+    <Pill
+      text={"Traveling"}
+      size={"Large"}
+      active={true}
+      color={"Purple"}
+      icon={"white"}
+      iconOffset
+      shadow
+      clickCallback={pillClick}
```
Comment (ZakkMan) [inline client/src/components/Pill/Pill.stories.js]: Let's call this something like `onClick`, to be inline with convention

File: client/src/components/Pill/Pill.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    // Default CSS class to apply to the Component
+    let classList = this.props.hasOwnProperty("active")
+      ? this.togglePillColor(this.props.active)
+      : this.assignStaticClassList();
+
+    this.state = {
+      classList: classList,
+      active: this.props.active,
+      selectId: props.selectId
+    };
```
[the remark this answers is not in the document]
Comment (clarknoah) [reply client/src/components/Pill/Pill.js]: When a user clicks on the component, it changes the state.

File: client/src/components/Pill/Pill.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    }
+  };
+
+  renderHeart = () => {
+    let iconClass = this.props.iconOffset ? "PillIcon PillIconOffset" : "PillIcon";
+    let renderHeart = this.props.icon.toLowerCase() === "pink" ? heartPink : heartWhite;
+    let img = <img src={renderHeart} className={iconClass}></img>;
+    let render = this.props.icon ? img : <div />;
+    return render;
+  };
```
[the remark this answers is not in the document]
Comment (clarknoah) [reply client/src/components/Pill/Pill.js]: I sure can, can I use this as a placeholder until he is done though?

**Pull request merged**